src/CommonDBTM.php

```diff
@@ -680,6 +680,9 @@ public function updateInDB($updates, $oldvalues = [])
                 unset($oldvalues[$field]);
             }
         }
+        if (count($tobeupdated) === 0) {
+            return true;
+        }
```

Comment:
This seems correct to me, but may have unexpected side effects. Indeed, with this change, an empty update does would trigger the `item_update` hook and the `post_updateItem()` method call. It would probably fix some edge cases, but may also break some plugins.

I agree with proposed change, but I think it should be done in the `main` branch.